libnd4j/include/ops/declarable/impl/DeclarableOp.cpp

```diff
@@ -697,39 +698,75 @@ sd::Status sd::ops::DeclarableOp::execute(Context *block) {
   return status;
 }
 
-void DeclarableOp::overwriteResult(Context &block, int outputIdx, NDArray *array) {
-  throw std::runtime_error("Overwrite result used!");
-  // block.pushNDArrayToVariableSpace(block.nodeId(), outputIdx, array);
-  /*
-  auto varSpace = block.getVariableSpace();
-  if (varSpace->hasVariable(block.getNodeId(), outputIdx)) {
+
+void DeclarableOp::overwriteResult(Context &block, int outputIdx, NDArray *array,bool remove) {
+  sd_printf("Pushing variable\n",0);
```

Comment:
shouldn't this be an sd_debug? Same applies to other usages of sd_printf and printIndexedBuffer in this file